Description: Remove restrictions on port
Bug-Kali: https://gitlab.com/kalilinux/internal/roadmap/-/issues/146
Forwarded: not-needed
Last-Update: 2021-04-12
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/include/net/sock.h
+++ b/include/net/sock.h
@@ -1397,8 +1397,8 @@ static inline int __sk_prot_rehash(struc
 /* About 10 seconds */
 #define SOCK_DESTROY_TIME (10*HZ)
 
-/* Sockets 0-1023 can't be bound to unless you are superuser */
-#define PROT_SOCK	1024
+/* All sockets for all */
+#define PROT_SOCK	0
 
 #define SHUTDOWN_MASK	3
 #define RCV_SHUTDOWN	1
